config/core/resources/route.yaml

```diff
@@ -45,6 +52,12 @@ spec:
   scope: Namespaced
   subresources:
     status: {}
+  conversion:
+    strategy: Webhook
+    webhookClientConfig:
+      service:
+        name: webhook
+        namespace: knative-serving
```

Comment:
Since we have had problems previously with the webhook not getting invoked. For the conversion webhook, what is the behavior if the webhook is not running? Do requests for all API versions fail, or just the non-storage version?

Reply:
> what is the behavior if the webhook is not running? 

Any operation that requires conversion will fail (ie. gets, creates, updates etc.)

> Do requests for all API versions fail, or just the non-storage version?

Thus if you're operating against the storage version the conversion webhook is not invoked.